src/Control/Monad/Bayes/Population.hs

```diff
@@ -269,7 +271,7 @@ popAvg f p = do
 
 -- | Applies a transformation to the inner monad.
 hoist ::
-  Monad n =>
```

Comment:
Are you still planning to replace this with MMorph?

Reply:
Yes, eventually. [`hoist`](https://hackage.haskell.org/package/mmorph-1.2.0/docs/Control-Monad-Morph.html#v:hoist) from `mmorph` brings the `Monad m` constraint, so this change should be fine. It's the `Monad n` constraint that is tricky.